### ISSUE TITLE
model doesn't converge 

### ISSUE THREAD

**Issue by vidyatshetty** [Microsoft/CNTK#3213]
hi,

I am training a speaker recognition model using TIMIT. when I train using 8samples per class and test using 2samples per class , the network converges and gives accuracy of 98% on training set and 45% on testing test. so i used data augmentation and increased the data size(30 samples per classs), when I train the model with increased number of samples it doesn't converge and it remains with metric = 75% even for 50,000 epochs. Kindly suggest is there something I am forgetting?
Thank you

**Comment by KeDengMS** [comment]
It depends on how you augment the data. If it's just repeating the same sample multiple times, it's essentially increasing learning rate which may lead to divergence.

**Comment by vidyatshetty** [comment]
@KeDengMS  I augmented the data by adding noise, stretch and roll, pitch shift. so for every 1 sample i generated 3 different samples.
When I increase the data for one class and keep the rest as it is the model converges but when I increase data for more than one class the network never converge. I quite don't understand what is happening. Please let me know if I am forgetting something.
Thanks

**Comment by KeDengMS** [comment]
First you need to decide whether it's overfitting or underfitting. Please try change learning rate to see if convergence changes, or change to a different learner. If none of those work, your model may not be able to represent the data augmentation, and you should try less randomness or less random sample. If you think that the augmentation is OK, then you may need to increase the model complexity to have more modeling capacity (a.k.a try to get it overfit first).

**Comment by vidyatshetty** [comment]
@KeDengMS  thank you for writing. it is overfitting. because I get accuracy of 97-98% on training set but only 50% accuracy on testing set. 
I tried changing the learning rate but it did not help in converging. 
As you said I will try to increase the model capacity for the augmented data and see if I achieve convergence(overfit first)
